constitution.tex

```diff
@@ -275,18 +275,23 @@ \subsection{Modifying the Constitution}
 	\item Receive priority for on-floor housing, or apply for On-Floor Status
 \end{enumerate}
 
-\asubsubsection{Expectations of Voting Members}
+\asubsubsection{Qualifications of Voting Members}
 \renewcommand{\theenumi}{\arabic{enumi}} % For this section, we want items to use letters
 
-The following requirements must be met by the beginning of the Intro Eval for an Active Member to be allowed to vote.
-Any of these requirements may be waived by the Evals Director or an E-Board Vote.
+For an Active Member to be allowed to vote in an Intro Eval, they must meet the following requirements by the beginning of the Intro Eval.
```

Comment:
Is the change to make Active Members who fail gatekeep keep voting rights in non-intro-eval settings intentional?

Reply:
If the intended result is to make the change noted above, then perhaps it would be good to specify that the only voting rights being modified are those at the Intro Eval. That would also change line 257 to be something like:

Active Members are eligible to vote at an Intro Eval if they meet the requirements defined in \ref{Qualifications of Voting Members}.

Reply:
It is intended, but I don't think there's currently any ambiguity. We could call them Introductory Voting Members instead if you think that's clearer.